src/resonanceReconstruction/rmatrix/test/rmatrix.test.cpp

```diff
@@ -12,7 +12,7 @@ using Spin = rmatrix::Spin;
 using ResonanceRange = endf::ResonanceRange;
 
 constexpr AtomicMass neutronMass = 1.008664 * daltons;
-constexpr ElectricalCharge elementaryCharge = 1.602e-19 * coulomb;
+constexpr ElectricalCharge elementaryCharge = dimwits::constant::elementaryCharge;
```

Comment:
What about the value above this?  Defining constants in a test clearly isn't ideal, hence the need for this one.

Reply:
True, but it's better than the 1.602e-19 under it :-)

Contrary to the elementary change, the neutron mass value is not defined in dimwits, I would have to import the constants package. The whole reason behind giving these values in the fromENDF function call was not having to do that.